test/verify/check-shell-multi-machine-key

```diff
@@ -70,17 +70,17 @@ class TestMultiMachineKeyAuth(MachineCase):
         self.browser.eval_js("loaded = false")
         self.browser.eval_js("""
             load = function (user) {{
-                var proc = cockpit.spawn([ 'ssh-add', '{0}' ], {{ pty: true, directory: user.home + '/.ssh' }})
-                    .stream(function(data) {{
+                const proc = cockpit.spawn([ 'ssh-add', '{0}' ], {{ pty: true, directory: user.home + '/.ssh' }});
```

Comment:
No need to store this in variable, but does not really matter.

Reply:
How do you mean? It's being used in the `.stream` handler, it refers to the original `cockpit.spawn()` return object. What you *can't* do (I learned from debugging) is

    const proc = cockpit.spawn().stream().then().catch()

and use *that* value -- at that time it is a transformed promise, which does not have an `.input()` function any more.

Reply:
Oh, it is being used for `input()` - missed that, nevermind :)